infra/gcp/main.tf

```diff
@@ -149,7 +149,7 @@ resource "google_container_node_pool" "vdc_preemptible_pool" {
   }
 
   node_config {
-    preemptible = true
+    spot = true
     machine_type = "n1-standard-4"
 
     labels = {
```

Comment:
We should also change the preemptible label and taint to reduce confusion.

Probably best to do that in three steps. Add a spot label to the nodes, add spot toleration to our code. Merge  a PR that node selects spot. Remove the preemptible label and add the spot taint.

We can do that in a separate PR series though.